server/src/middleware/file-upload.interceptor.ts

```diff
@@ -141,6 +143,14 @@ export class FileUploadInterceptor implements NestInterceptor {
 
   private handleFile(request: AuthRequest, file: Express.Multer.File, callback: Callback<Partial<ImmichFile>>) {
     (file as ImmichMulterFile).uuid = randomUUID();
+
+    request.on('error', (error) => {
+      if (error.message === 'aborted') {
+        this.handleAbortedRequest(request, file);
+        return;
+      }
+    });
```

Comment:
This should call `assetService.onUploadError(...)` or similar. Can you move the implementation to that service and add a test for it?